### PR TITLE
Update doc of `pads` attribute for conv and pool ops

### PR DESCRIPTION
**Pull request by bddppq**

### CODE REVIEW

File: onnx/defs/nn/defs.cc [hunk 1]
```diff
@@ -182,13 +176,13 @@ and computes the output.)DOC";
                         "stride along each axis.",
                         AttrType::INTS);
             schema.Attr("pads",
-                        "Padding along each axis, can take the value 0 (False) or non 0 (True)",
+                        "The number of zeros padding added to each axis.",
```
Comment (dzhulgakov) [inline onnx/defs/nn/defs.cc]: can we clarify explicitly that it's symmetrical padding? (or did we land on asymetrical)?

Comment (ebarsoum) [reply onnx/defs/nn/defs.cc]: @dzhulgakov  what you mean adding SAME/VALID setting? I assume that will replace current setting and not in addition, right?

When I added 'pads' attribute initially, I meant to have 1/0 per dimension, 1 mean SAME and 0 mean VALID. However, I didn't mean upper and lower per dimension. I have no problem to use SAME/VALID settings. I don't won't end user to specify asymmetric padding, if needed they can use ONNX.pad OP.

Comment (dzhulgakov) [reply onnx/defs/nn/defs.cc]: So I think there's actually discrepancy here. Both C2 and PyTorch just specify the list of padding in pixels for each dimension.

Specifying the VALID/SAME for each dimension independently is a bit unnatural and I don't think any framework supports it except for CNTK. Do you see a lot of use cases for it actually?

I think the reasonable support would be to have either:
1) arbitrary padding in pixels per dimension. This is the most general and matches for example TensorRT/PyTorch/C2.
2) one global setting of SAME to keep compatibility with TF.

That's the kind of setup we discussed in person last Wednesday on the sync. What do you think about it?

**Comment by bddppq** [comment]
Continue in #127

**Pull request closed without merging**